core/evm/sputnikvm/gallectic_adapter.go

```diff
@@ -80,7 +80,7 @@ func (ga *GallacticAdapter) updateStorage(address common.Address, key *big.Int,
 	ga.Cache.SetStorage(addr, wKey, wValue)
 }
 
-func (ga *GallacticAdapter) getStorage(address common.Address, key *big.Int) (*big.Int, error) {
+func (ga *GallacticAdapter) getStorage(address common.Address, key *big.Int) *big.Int {
 	addr := fromEthAddress(address, true)
 	wKey := binary.LeftPadWord256(key.Bytes())
 	wValue, err := ga.Cache.GetStorage(addr, wKey)
```

Comment:
What happen if there is a valid data in storage but we get error in reading? In this code in case of error, u set storage to zero. I think this isn't logical. I think get method is for reading and any writing should be performed by set methods. so, I think returning error is more logical.

Reply:
Good point.